src/pybind/rados/rados.pyx

```diff
@@ -228,6 +234,8 @@ IF UNAME_SYSNAME == "FreeBSD":
         errno.EISCONN   : IsConnected,
         errno.EINVAL    : InvalidArgumentError,
         errno.ENOTCONN  : NotConnected,
+        errno.ERANGE    : ObjectVerMismatch,
```

Comment:
my only concern is that there are some other error cases where `-ERANGE` and `-EOVERFLOW` are also returned. i don't have a better proposal here, probably we could just document the behavior as the docstring of `assert_version()` that

- `ERANGE` is returned if the specified version is less than the object's version
- `EOVERFLOW` is returned if  the specified version is greater than the object's version

?